src/integer_mod_q/mat_zq/from.rs

```diff
@@ -126,6 +126,37 @@ impl<Mod: Into<Modulus>> From<(&MatZ, Mod)> for MatZq {
     }
 }
 
+impl<Mod: Into<Modulus>> From<(MatZ, Mod)> for MatZq {
+    /// Creates a [`MatZq`] from a [`MatZ`] and a [`Modulus`].
+    ///
+    /// Parameters:
+    /// - `matrix`: the matrix from which the entries are taken
+    /// - `modulus`: the modulus of the matrix
+    ///
+    /// Returns the new matrix.
```

Comment:
Yes all these inconsistencies in the doc comments bothered me too, but in the last PRs I made to many refactorings when I just wanted to implement one new function. So I decided to do them all in a separate PR this time. But if you want I can do those directly.

Reply:
The first line should be a brief explanation what the function does. It's ok if they are not consistent. It's more important that they are precise and tell the developer what the function does.
The comment `Return the new matrix` whoever wrote it and whoever allowed it to make it to dev, didn't have a look at that, because it is not descriptive or explanatory.
Hence, I wouldn't introduce more bad examples on purpose but rather get rid of them where I can if I have the opportunity to do it.

Reply:
I have now corrected all similar docs I have found in the math crate.